parts/plugins/x-apache.py

```diff
@@ -188,7 +188,7 @@ def build(self):
 
         self.run(
             ['make', '-j{}'.format(
-                snapcraft.common.get_parallel_build_count())],
+                self.project.parallel_build_count)],
```

Comment:
Sorry about this-- I should have pushed up my fix before releasing 2.8. But you're missing another one of these in here for building the modules.